Funcky.Test/OptionTest.cs

```diff
@@ -464,6 +464,21 @@ public void InspectDoesNotExecuteSideEffectWhenOptionIsNone()
             Assert.Equal(option, inspectedOption);
         }
 
+        [Fact]
+        public void GivenAFunctionWithADefaultOptionParameterGivesANoneWhenNoValueGiven()
+        {
+            MethodWithDefaultOptionParameter()
+                .Match(none: () => Assert.True(true), some: s => Assert.True(false));
+
+            MethodWithDefaultOptionParameter(Option.Some("value"))
+                .Match(none: () => Assert.True(false), some: s => Assert.Equal("value", s));
+        }
+
+        private Option<string> MethodWithDefaultOptionParameter(Option<string> value = default)
```

Comment:
Use expression body

Reply:
Fixed in 9f28d8edf8a3e703fde28b22e3350e86d2fe6aac